bot/cogs/token_remover.py

```diff
@@ -16,6 +16,10 @@
 
 log = logging.getLogger(__name__)
 
+LOG_MESSAGE = (
+    "Censored a seemingly valid token sent by {author} (`{author_id}`) in {channel},"
```

Comment:
Good catch. Also, the channel name is already being formatted (it uses `channel.mention`).

Reply:
The bot logs will only include `<#ID>` which is less useful than a name with the id but I guess that's worth the tradeoff because then it'd result in the name being logged twice in the channel

Reply:
Sorry, you're right. I think I'll leave it alone though. I don't see a clean way to have separate messages and it doesn't feel to be worth the trouble.